glgen.sh

```diff
@@ -64,9 +64,14 @@ cat > "$OUTDIR/$BASE.h" << EOF
 #include <stdbool.h>
 #include <wlr/config.h>
 
-#if !WLR_HAS_X11_BACKEND && !WLR_HAS_XWAYLAND && !defined MESA_EGL_NO_X11_HEADERS
+#if !WLR_HAS_X11_BACKEND && !WLR_HAS_XWAYLAND
+#ifndef MESA_EGL_NO_X11_HEADERS
```

Comment:
Why do we need these `#ifndef`s?

Reply:
Ah, because multiple of these headers might be included at te same time.

Reply:
Otherwise redefining one of them would make the build fails.